forge/csrc/passes/commute_utils.cpp

```diff
@@ -539,7 +539,11 @@ bool commute_through_concat(
         concat_golden_transform.attr[concat_dim >= 0 ? concat_dim : concat_dim + concat_golden_transform.attr.size()] =
             concat_output_len;
     op->add_golden_transform(concat_golden_transform);
-    op->overwrite_op_attrs(attr);
+    std::vector<graphlib::OpType::Attr> concat_attr;
+    concat_attr.push_back(new_dim);
```

Comment:
@meenakshiramanathan1 we have this overwrite_op_attrs on many places in this part of the stack. Can you create an issue to transition to named attributes? This will bite us little by little if we solve only partial cases like this one. I would ask you to follow up on this PR with another one fixing this throughout the commute_utils. Does this sound reasonable?

Reply:
Sure @dgolubovicTT , I will create an issue and follow up with another PR with changes transitioning to named attributes.